edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -94,3 +94,23 @@ Feature: patron tests
     Then status 200
     Then match response.totalHolds == 1
     And match response.holds[0].item.itemId == itemId
+
+   Scenario: Create a "first available item" request via an external form.
```

Comment:
There is no form involved in the back end API, I suggest not including that in the description.

The important part of this scenario is that there is only a single item associated with the instance being requested.

```suggestion
   Scenario: Instance level request is associated with only item
```